lib/3scale/backend/transactor.rb

```diff
@@ -182,6 +196,19 @@ def preload_usage_limits(application, no_body_enabled, usage_in_params)
         end
       end
 
+      def filter_usages_with_0(transactions)
+        # There are plenty of existing tests using both a string and a symbol
+        # when accessing the usage.
+        transactions.delete_if do |_idx, tx|
+          (usage = tx['usage'.freeze] || tx[:usage]) or next
```

Comment:
```ruby
puts 'The parens are not needed' or
  puts 'because the natural language operators have the lowest precendence' or
  puts 'even lower than assignment' and
  nil.call.causes_no_method_error and
  no_worries.it_is_never_executed_anyway and
  (ツ) or
  puts "ok, well, whatever!"
```